packages/tools/client-debugger/client-debugger-view/src/ContainerHistoryLog.tsx

```diff
@@ -113,10 +113,10 @@ export function ContainerHistoryLog(props: ContainerHistoryLogProps): React.Reac
 								backgroundColor: getBackgroundColorForState(item.newState),
 							}}
 						>
-							<TableCell>
+							<TableHeaderCell>
```

Comment:
This is a body cell - we probably shouldn't use the header cell component here. It will likely annotate the output as being a header cell when it really isn't. We can wrap the inner contents in a horizontal stack - that might fix things for us (or we can apply styling as necessary).

Reply:
I thought changing to `<TableHeaderCell>` was a "clever" way to avoid applying styling to the container state 😅 . Changed by applying styling to `<TableCell>`.

Reply:
Yeah, if it didn't affect the generated HTML it would make sense, but I suspect that it does. In theory this could affect screen-reader behavior, etc., so we have to be careful.